crates/sp-domains/src/lib.rs

```diff
@@ -218,7 +218,6 @@ pub struct ProofOfElection<DomainHash> {
 }
 
 impl<DomainHash: Default> ProofOfElection<DomainHash> {
-    #[cfg(feature = "std")]
```

Comment:
I don't think this is necessary for no_std, you can add the `runtime-benchmarks` feature into sp-domains as well.